drivers/SPI.h

```diff
@@ -143,6 +143,14 @@ class SPI : private NonCopyable<SPI> {
      */
     virtual void unlock(void);
 
+    /** SPI block read dummy data
+      * SPI requires master to send dummy data, in order to perform read operation.
+      * Dummy bytes can be different for devices. Example SD Card require 0xFF.
```

Comment:
Edits in bold:
**Different devices may require different dummy byte values.** **For** **e**xample: **A** SD Card require**s** **dummy bytes to be** 0xFF.